app/assets/stylesheets/new_board/_story.scss

```diff
@@ -4,6 +4,26 @@
     border-bottom: 1px solid $story-border-bottom-color;
     border-top: 1px solid $story-border-top-color;
 
+    &__task-form {
+      @extend .clearfix;
+    }
+
+    &__add-task-button {
+      @extend .btn;
+      @extend .btn-default;
+      @extend .btn-xs;
+      @extend .pull-right;
+      margin-top: 10px;
+    }
+
+    .checkbox {
```

Comment:
Why isn't this called `&__add-task-checkbox`?

Reply:
I agree with you, but I change the name for `&__task` because this is relative about the component Task! thanks